scipy/stats/_stats_py.py

```diff
@@ -10261,7 +10479,6 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     """
     return _cdf_distance(1, u_values, v_values, u_weights, v_weights)
```

Comment:
Please also consider whether the new input validation there would still be appropriate. (Perhaps it is performed in _cdf_distance, but if not, we should perform the checks and add tests of the input validation.)

Reply:
I think it would be appropriate. I have not changed the code in `_stats_py.py`. I have only modified the docs and tests.